### PR TITLE
scheduler: delete docs related to equivalence cache from scheduler

### PR DESCRIPTION
**Pull request by sanposhiho**
#### What type of PR is this?

/kind cleanup
/kind documentation

#### What this PR does / why we need it:

delete comments about equivalence cache.

---

I don't know about equivalence cache now and I found that a cache of that name existed in the past.

If it still exists somewhere or was renamed to something else, please let me know. It will be helpful for me🙏 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/scheduler/eventhandlers.go [hunk 1]
```diff
@@ -415,9 +400,6 @@ func addAllEventHandlers(
 				)
 			}
 		case framework.Service:
-			// ServiceAffinity: affected by the selector of the service is updated.
-			// Also, if new service is added, equivalence cache will also become invalid since
-			// existing pods may be "captured" by this service and change this predicate result.
```
Comment (Huang-Wei) [inline pkg/scheduler/eventhandlers.go:420]: Let's keep L418.

**Review by Huang-Wei — COMMENTED** [review]
One nit. LGTM otherwise.

File: pkg/scheduler/eventhandlers.go (hunk 1 above)
Comment (sanposhiho) [reply pkg/scheduler/eventhandlers.go]: thanks, fixed 👍

**Pull request merged**